x/ccv/provider/keeper/double_vote_test.go

```diff
@@ -0,0 +1,311 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	testutil "github.com/cosmos/interchain-security/v2/testutil/crypto"
+	testkeeper "github.com/cosmos/interchain-security/v2/testutil/keeper"
+	"github.com/stretchr/testify/require"
+	tmencoding "github.com/tendermint/tendermint/crypto/encoding"
+	tmprotocrypto "github.com/tendermint/tendermint/proto/tendermint/crypto"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+func TestVerifyDoubleVotingEvidence(t *testing.T) {
+	keeper, ctx, ctrl, _ := testkeeper.GetProviderKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
+	defer ctrl.Finish()
+
+	chainID := consumer
+
+	signer1 := tmtypes.NewMockPV()
+	signer2 := tmtypes.NewMockPV()
+
+	val1 := tmtypes.NewValidator(signer1.PrivKey.PubKey(), 1)
+	val2 := tmtypes.NewValidator(signer2.PrivKey.PubKey(), 1)
+
+	valSet := tmtypes.NewValidatorSet([]*tmtypes.Validator{val1, val2})
+
+	blockID1 := testutil.MakeBlockID([]byte("blockhash"), 1000, []byte("partshash"))
+	blockID2 := testutil.MakeBlockID([]byte("blockhash2"), 1000, []byte("partshash"))
+
+	ctx = ctx.WithBlockTime(time.Now())
+	oldTime := ctx.BlockTime().Add(-505 * time.Hour)
```

Comment:
Would it be possible to reuse some constant here (e.g., `ctx.ConsensusParams().Evidence.MaxAgeDuration`) instead of `-505` to make it clear that we go enough back in time for the evidence to be too old?